src/gtpu/encap.c

```diff
@@ -651,7 +650,7 @@ int check_urr(struct pdr *pdr, u64 vol, u64 vol_mbqe, bool uplink) {
                 ret = -1;
```

Comment:
why no `ret = -1;` if netlink_send() <0 and no `++pdr->dl_drop_cnt;` if unix_sock_send() < 0 ?